internal/venice-common/src/main/java/com/linkedin/venice/meta/DataReplicationPolicy.java

```diff
@@ -7,6 +7,7 @@
 
 /**
  * Enums of the policies used to decide how real-time samza data is replicated.
+ * Note: Data replication policy does not play any role for AA enabled hybrid stores.
```

Comment:
nit: The DRP often cause some confusions on me so can you please add more notes here? So, the default DRP for AA store would be `NON_AGGREGATE`? And do we have validation if AA store is updated with `AGGREGATE` as DRP?
BTW, can we also update some internal runbooks to reflect this?

Reply:
> The DRP often cause some confusions on me so can you please add more notes here? So, the default DRP for AA store would be NON_AGGREGATE? 
yes. But it will not have any effect

>And do we have validation if AA store is updated with AGGREGATE as DRP?
If store is AA enabled DRP is of no use. I don't follow what you are saying here. Could you add more details? Thanks


>BTW, can we also update some internal runbooks to reflect this?
sure

Reply:
Done

Reply:
Understood. I thought DPR is required to be  `NON_AGGREGATE` when AA is on so we need to do some validation in case it's set to `AGGREGATE`. Since it doesn't play a role at all, then it's not needed